File: test/e2etestsmain5/pipelines/Workflow-Automated-GITHUB-1664179694988/code/workflowautomatedgithub1664179694988/graph/Filter_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from workflowautomatedgithub1664179694988.config.ConfigStore import *
from workflowautomatedgithub1664179694988.udfs.UDFs import *

def Filter_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(((col("customer_id") > lit(10)) & col("first_name").like("%G%")))

File: test/e2etestsmain5_external/pipelines/Workflow-Automated-GITHUB-1666175154627/code/src/main/scala/io/prophecy/pipelines/workflowautomatedgithub1666175154627/graph/Filter_1.scala
package io.prophecy.pipelines.workflowautomatedgithub1666175154627.graph

import io.prophecy.libs._
import io.prophecy.pipelines.workflowautomatedgithub1666175154627.config.ConfigStore._
import io.prophecy.pipelines.workflowautomatedgithub1666175154627.udfs.UDFs._
import io.prophecy.pipelines.workflowautomatedgithub1666175154627.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.filter((col("customer_id") > lit(10)).and(col("first_name").like("%G%")))

}
